Documentation/MigrateToComposer/CoWorking.rst

```diff
@@ -8,38 +7,38 @@ Co-working
 Add to version control system
 =============================
 
-If you use a version control system like Git, it's important to add
-both files, the :file:`composer.json` and :file:`composer.lock` (which
-automatically was created during the previous steps).
+If you use a version control system such as Git (and you really should!), it is
+important to add both files :file:`composer.json` and :file:`composer.lock`
+(which were created automatically during the previous steps). The
+:file:`composer.lock` file keeps track of the exact versions that are installed,
+so that you are on the same versions as your co-workers (and when deploying to
+the live system).
 
-On the other side, some files and folders, which are added by
-composer, should be excluded:
+On the other side, some files and folders added by Composer should be excluded:
```

Comment:
I think you can leave out "On the other side" and "On the other hand" might be better